### ISSUE TITLE
momentumbots api returns little results

### ISSUE THREAD

**Issue by verabehr** [VolumeFi/paloma#221]
The momentumbots API that lists the search results of momentum tokens returns little results and at times none. Using this ticket to capture some searches. Note that this could either be expected due to market conditions or a bug somewhere in the code. 

even searching for https://momentumbots.palomachain.com/?dex=uniswap_v2&lag_return=0&daily_volume=0&vol_30=0&market_cap=0 only returns one token and at times none

Separately or related 

```
127.0.0.1 - - [18/May/2023 19:40:26] "POST /get_high_returns HTTP/1.0" 500 767
Traceback (most recent call last):
  File "/root/.cache/pypoetry/virtualenvs/trading-bots-Fx44JTns-py3.10/lib/python3.10/site-packages/bottle.py", line 876, in _handle
    return route.call(**args)
  File "/root/.cache/pypoetry/virtualenvs/trading-bots-Fx44JTns-py3.10/lib/python3.10/site-packages/sentry_sdk/integrations/bottle.py", line 140, in wrapped_callback
    raise exception
  File "/root/.cache/pypoetry/virtualenvs/trading-bots-Fx44JTns-py3.10/lib/python3.10/site-packages/sentry_sdk/integrations/bottle.py", line 130, in wrapped_callback
    res = prepared_callback(*args, **kwargs)
  File "/root/.cache/pypoetry/virtualenvs/trading-bots-Fx44JTns-py3.10/lib/python3.10/site-packages/bottle.py", line 1759, in wrapper
    rv = callback(*a, **ka)
  File "/root/app/momentum_cache_refresh/app.py", line 25, in get_high_returns
    df = momentum_scanner_intraday.get_high_returns(
  File "/root/app/momentum_cache_refresh/momentum_scanner_intraday.py", line 73, in get_high_returns
    vols = gecko.exchanges(dex)
  File "/root/app/momentum_cache_refresh/gecko.py", line 29, in exchanges
    for ticker in get("exchanges", dex)["tickers"]
KeyError: 'tickers'
```

That last bit means it's straight from coingecko's API.

**Comment by fieldtheory123** [comment]
This is fixed by https://github.com/VolumeFi/trading-bots/pull/22

**Comment by taariq** [comment]
fixed with #22
